assets/_sass/pages/_archive.scss

```diff
@@ -449,4 +449,8 @@
     max-width: 30px;
     background-color: transparent;
   }
+
```

Comment:
Can you add a comment explaining why we're doing this? Something like "Until more analysis content is added to the site, the filter selection has been visually hidden"